src/net/sourceforge/kolmafia/persistence/TCRSDatabase.java

```diff
@@ -619,17 +620,17 @@ public static void setEffectName(final Integer itemId, String name) {
   }
 
   private static void addEffectSource(
-      final String itemName, final int usage, final String effectName) {
+      final String itemName, final ConsumptionType usage, final String effectName) {
     int effectId = EffectDatabase.getEffectId(effectName);
     if (effectId == -1) {
       return;
     }
     String verb =
```

Comment:
I've not mentioned converting to modern switch statements, but this would be a perfect use for a switch expression.

Reply:
I avoided switching to expressions to keep the difference lower, but I've converted this and another ternary if.